apps/sms/js/thread_list_ui.js

```diff
@@ -23,6 +23,8 @@ function createBdiNode(content) {
 }
 
 var ThreadListUI = {
+  readyDeferred: Utils.Promise.defer(),
```

Comment:
I hate that it's here, but I can think of any better way to make it resettable for tests